docs/standard/serialization/binaryformatter-migration-guide/wpf-applications.md

```diff
@@ -0,0 +1,104 @@
+
+### Clipboard  
+
+In WPF application if you are using a clipboard to copy or paste the data into the application or Set/Get data from a DataObject or using drag/drop. 
+
+If the data format ([DataFromats](https://learn.microsoft.com/en-us/dotnet/api/system.windows.dataformats?view=windowsdesktop-8.0)) is part of the following list, it will not require BinaryFormatter to handle: 
+- `Stream` 
+- `Text`, `Rtf`, `OemText`, `CommaSeparatedValue` 
+- `UnicodeText` 
+- `FileDrop` 
+- `FileName` 
+- `BitmapSource` 
+
+Apart from these formats, if the type is serializable and not included in the above list of types handled by the new implementation, `Binaryformatter` will be used for handling. 
+
+In scenarios where `Binaryformatter` is utilized, its removal will result in developers encountering an exception indicating that `Binaryformatter` has been removed. 
+
+You can refer to the function where we have used `Binaryformatter` as fallback to read/save object to handle:  [SaveObjectToHandle](https://github.com/dotnet/wpf/blob/0354a597996adae43b12efc72bd705f76d4ba497/src/Microsoft.DotNet.Wpf/src/PresentationCore/System/Windows/dataobject.cs#L1677) and  [ReadObjectFromHandle](https://github.com/dotnet/wpf/blob/0354a597996adae43b12efc72bd705f76d4ba497/src/Microsoft.DotNet.Wpf/src/PresentationCore/System/Windows/dataobject.cs#L3051) 
+
+Additionally, if you encounter an exception stating, 'Data stored in clipboard is invalid,' it may be because `Binaryformatter` was used to save the data. 
+
+Ref: [DataObject.cs](https://github.com/dotnet/wpf/blob/4e977f5fe8c73094ee5826dbfa7a0f37c3bf0e33/src/Microsoft.DotNet.Wpf/src/PresentationCore/System/Windows/dataobject.cs) 
+
+ 
+
+### Drag/Drop 
+
+In drag/drop any UI element or Content element can participate, for an event is raised which is defined in [DragDrop.cs](https://github.com/dotnet/wpf/blob/0354a597996adae43b12efc72bd705f76d4ba497/src/Microsoft.DotNet.Wpf/src/PresentationCore/System/Windows/DragDrop.cs). In the Drag/Drop operation a DataObject is used to store the data and to Get/Set data we call `GetData()`/`SetData()`. In this scenario if the type involved is not intrinsically handled during serialization/deserialization, `Binaryformatter` is used. 
+
+With `Binaryformatter` removed, developers will now encounter an exception indicating that `Binaryformatter` has been removed. 
+	 
+
+### Journaling 
+
+In the case when we need to store or load a state while managing the navigation history in WPF. 
+
+To load/save we call [LoadSubStreams](https://github.com/dotnet/wpf/blob/0354a597996adae43b12efc72bd705f76d4ba497/src/Microsoft.DotNet.Wpf/src/PresentationFramework/MS/Internal/DataStreams.cs#L244)/ [SaveSubStreams](https://github.com/dotnet/wpf/blob/0354a597996adae43b12efc72bd705f76d4ba497/src/Microsoft.DotNet.Wpf/src/PresentationFramework/MS/Internal/DataStreams.cs#L86) of DataStream class. If the element used in not part of know type handled by the new implementation, it will use `Binaryformatter`. 
```

Comment:
Is there some sort of user action that triggers the call to load/save? That might be good to call out if so.

Reply:
User can call save/load just by navigating to/back/forward, as at each step SaveState function of JournalEntry is called which further trigger SaveSubStreams/LoadSubStream.
I used this scenario for testing, there could be other scenarios too.